src/SprykerSdk/Acp/Schema/SchemaExtender.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Acp\Schema;
+
+use Symfony\Component\Process\Process;
+use Transfer\CreateDefaultEndpointsRequestTransfer;
+use Transfer\CreateDefaultEndpointsResponseTransfer;
+use Transfer\MessageTransfer;
+
+class SchemaExtender implements SchemaExtenderInterface
+{
+    /**
+     * @var \SprykerSdk\Acp\Schema\SchemaConverterInterface
+     */
+    protected SchemaConverterInterface $schemaConverter;
+
+    /**
+     * @var \SprykerSdk\Acp\Schema\SchemaWriterInterface
+     */
+    protected SchemaWriterInterface $schemaWriter;
+
+    /**
+     * @var string
+     */
+    protected string $acpRootPath;
+
+    /**
+     * @param \SprykerSdk\Acp\Schema\SchemaConverterInterface $schemaConverter
+     * @param \SprykerSdk\Acp\Schema\SchemaWriterInterface $schemaWriter
+     * @param string $acpRootPath
+     */
+    public function __construct(
+        SchemaConverterInterface $schemaConverter,
+        SchemaWriterInterface $schemaWriter,
+        string $acpRootPath
+    ) {
+        $this->schemaConverter = $schemaConverter;
+        $this->schemaWriter = $schemaWriter;
+        $this->acpRootPath = $acpRootPath;
+    }
+
+    /**
+     * @param \Transfer\CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer
+     *
+     * @return \Transfer\CreateDefaultEndpointsResponseTransfer
+     */
+    public function extendProjectSchema(CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer): CreateDefaultEndpointsResponseTransfer
+    {
+        $createDefaultEndpointsResponseTransfer = $this->addDefaultEndpointsToSchema($createDefaultEndpointsRequestTransfer);
+
+        $configurationSchemaGeneratedMessage = $this->convertConfigurationToSchema($createDefaultEndpointsRequestTransfer);
+
+        if ($configurationSchemaGeneratedMessage) {
+            $createDefaultEndpointsResponseTransfer->addError($configurationSchemaGeneratedMessage);
+        }
+
+        return $createDefaultEndpointsResponseTransfer;
+    }
+
+    /**
+     * @param \Transfer\CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer
+     *
+     * @return \Transfer\CreateDefaultEndpointsResponseTransfer
+     */
+    protected function addDefaultEndpointsToSchema(
+        CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer
+    ): CreateDefaultEndpointsResponseTransfer {
+        $result = new CreateDefaultEndpointsResponseTransfer();
+
+        $process = new Process([
+            $this->acpRootPath . '/vendor/bin/syncapi',
```

Comment:
This will not work outside of the ACP SDK sandbox.
From a clean "app" workflow with spryker-sdk (pay attention to the paths)
```
mads@xps:~/projects/app-demo$ php vendor/bin/acp app:default-endpoints:create --add-local
Updating openapi schema failed, error output: sh: 1: exec: /home/mads/projects/app-demo/vendor/spryker-sdk/acp/vendor/bin/syncapi: not found

Updating openapi schema with configuration schema failed, error output: sh: 1: exec: /home/mads/projects/app-demo/vendor/spryker-sdk/acp/vendor/bin/syncapi: not found
```

[This global from composer](https://getcomposer.org/doc/articles/vendor-binaries.md#finding-the-composer-bin-dir-from-a-binary) might be an option instead, but only for newer versions of composer.

Reply:
Implemented a solution using both that global and a fallback value based on bootstrap.php constant